bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -411,19 +396,19 @@ func (s *ksServer) InstallIstio(ctx context.Context, req CreateRequest) error {
 		return err
 	}
 
-	err = CreateResourceFromFile(config, path.Join(regPath, "../dependencies/istio/install/crds.yaml"))
+	err = utils.CreateResourceFromFile(config, path.Join(regPath, "../dependencies/istio/install/crds.yaml"))
 	if err != nil {
 		log.Errorf("Failed to create istio CRD: %v", err)
 		return err
 	}
-	err = CreateResourceFromFile(config, path.Join(regPath, "../dependencies/istio/install/istio-noauth.yaml"))
+	err = utils.CreateResourceFromFile(config, path.Join(regPath, "../dependencies/istio/install/istio-noauth.yaml"))
 	if err != nil {
 		log.Errorf("Failed to create istio manifest: %v", err)
 		return err
 	}
 	//TODO should be a cli parameter
```

Comment:
this has been eliminated in a later version of kustomize that would be a follow-on PR after yours. These resources are now in manifests.